undertow/src/main/java/org/wildfly/extension/undertow/HostDefinition.java

```diff
@@ -89,12 +89,20 @@
             .setDefaultValue(ModelNode.TRUE)
             .setAllowExpression(true)
             .build();
+    static final SimpleAttributeDefinition ACTIVE_REQUEST_TRACKING_ENABLED = new SimpleAttributeDefinitionBuilder(Constants.ACTIVE_REQUEST_TRACKING_ENABLED, ModelType.BOOLEAN, true)
+            .setRestartAllServices()
+            .setDefaultValue(ModelNode.FALSE)
+            .setAllowExpression(true)
+            .build();
 
     static final HostDefinition INSTANCE = new HostDefinition();
-    private static final Collection<AttributeDefinition> ATTRIBUTES = Collections.unmodifiableCollection(Arrays.asList(ALIAS, DEFAULT_WEB_MODULE, DEFAULT_RESPONSE_CODE, DISABLE_CONSOLE_REDIRECT, QUEUE_REQUESTS_ON_START));
+    private static final Collection<AttributeDefinition> ATTRIBUTES = Collections.unmodifiableCollection(
+            Arrays.asList(ALIAS, DEFAULT_WEB_MODULE, DEFAULT_RESPONSE_CODE, DISABLE_CONSOLE_REDIRECT,
+                    QUEUE_REQUESTS_ON_START, ACTIVE_REQUEST_TRACKING_ENABLED));
     private static final List<? extends PersistentResourceDefinition> CHILDREN = Collections.unmodifiableList(Arrays.asList(
             LocationDefinition.INSTANCE,
             AccessLogDefinition.INSTANCE,
+            ActiveRequestTrackerDefinition.INSTANCE,
```

Comment:
This new resource requires a transformer to handle the case where this resource is defined and the model needs to be transformed for a host using an older version of the model, which has no such resource registered.